src/components/constants/display-sections.js

```diff
@@ -1 +1,3 @@
 export const CHALLENGES_SCREEN = 'Challenges';
+export const DUMMY_PROFILE = 'dummyProfile.png';
+export const IMAGE_URL = process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL;
```

Comment:
This seems like a long way to do a simple thing 😅 
Import something from env variables in one file, export it, re-import again in the same file without doing anything with it.

We're sort of re-declaring it, which is not really necessary here. What do you think?

I was saying that in the other file itself you can define it as `const IMAGE_URL = process.env.NEXT_PUBLIC_GITHUB_IMAGE_URL` and go ahead

Reply:
You are right😅, I will make the changes right away